.github/workflows/integration-tests.yml

```
@@ -7,14 +7,18 @@ env:
 
 jobs:
   test-harbor-1-10-2:
```

This version reference can be dropped:
```suggestion
  test-harbor:
```